server/plugin/plugin_test.go

```diff
@@ -20,17 +20,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func setupTestPlugin(t *testing.T, api *plugintest.API, store *mockstore.Store, siteURL string) *MatterpollPlugin {
-	defaultServerLocale := "en"
+func setupTestPlugin(t *testing.T, api *plugintest.API, store *mockstore.Store) *MatterpollPlugin {
```

Comment:
I don't know why we would pass a `siteURL` here. Removing it makes tests much easier.